modules/data.atmosphere/R/met.process.R

```diff
@@ -459,13 +459,12 @@ browndog.met <- function(browndog, source, site, start_date, end_date, model, di
                     "<model>", model, "</model>", 
                     "</input>")
   
-  userpass <- paste(browndog$username, browndog$password, sep = ":")
-  curloptions <- list(userpwd = userpass, httpauth = 1L, followlocation = TRUE)
-  result <- RCurl::postForm(paste0(browndog$url, formatname, "/"),
-                     fileData = RCurl::fileUpload("pecan.xml", xmldata, "text/xml"), .opts = curloptions)
-  url <- gsub(".*<a.*>(.*)</a>.*", "\\1", result)
-  PEcAn.logger::logger.info("browndog download url :", url)
-  downloadedfile <- PEcAn.utils::download.url(url, outputfile, 600, curloptions)
+  pecanxml <- tempfile("pecan.xml")
+  write(xmldata, pecanxml)
+  result.url <- BrownDog::convert_file(browndog$url, pecanxml, formatname, "./", browndog$token, download=FALSE)
```

Comment:
make sure to update the config.example.php
is ./ the current folder? if so please use a temp folder, since you might not always be able to write to the current folder.

Reply:
which config.example.php? didn't find it. 

is ./ the current folder?  -- yes, but it's not used, since download is false.